astropy/wcs/low_level_api.py

```diff
@@ -0,0 +1,159 @@
+import abc
+
+import numpy as np
+
+__all__ = ['BaseLowLevelWCS']
+
+
+class BaseLowLevelWCS(metaclass=abc.ABCMeta):
+    """
+    Abstract base class for the low-level WCS interface described in APE 14
+    (https://doi.org/10.5281/zenodo.1188875)
```

Comment:
Is this the right "canonical" link, as opposed to linking to the APE repo?  I could see it either way but thought I might ask...